Multiplatform/InfoPlist.xcstrings

```diff
@@ -260,6 +296,12 @@
             "state" : "translated",
             "value" : "Rechercher dans toutes les bibliothèques"
           }
+        "ru" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Искать по всем библиотекам"
+          }
+        },
```

Comment:
Did you use Xcode for this or did you open the file in a text editor and copy the values? Because there are multiple of these formatting errors that probably wouldn't have occurred if this was done by a machine.